src/modules/Elsa.Workflows.Core/Services/WorkflowStateExtractor.cs

```diff
@@ -96,10 +96,10 @@ private void ApplyProperties(WorkflowState state, WorkflowExecutionContext workf
             workflowExecutionContext.Properties[property.Key] = property.Value;
     }
 
-    private static void ApplyActivityExecutionContexts(WorkflowState state, WorkflowExecutionContext workflowExecutionContext)
+    private static async Task ApplyActivityExecutionContexts(WorkflowState state, WorkflowExecutionContext workflowExecutionContext)
```

Comment:
Please mind the naming convention to apply the `Async` suffix when changing methods into async.